scripts/st2bootstrap-el7.sh

```diff
@@ -111,6 +111,31 @@ install_yum_utils() {
   sudo yum install -y yum-utils
 }
 
+function get_package_url() {
+  # Retrieve direct package URL for the provided dev build, subtype and package name regex.
+  DEV_BUILD=$1 # Repo name and build number - <repo name>/<build_num> (e.g. st2/5646)
+  DISTRO=$2  # Distro name (e.g. trusty,xenial,el6,el7)
+  PACKAGE_NAME_REGEX=$3
+
+  PACKAGES_METADATA=$(curl -Ss -q https://circleci.com/api/v1.1/project/github/StackStorm/${DEV_BUILD}/artifacts)
+
+  if [ -z "${PACKAGES_METADATA}" ]; then
+      echo "Failed to retrieve packages metadata from https://circleci.com/api/v1.1/project/github/StackStorm/${DEV_BUILD}/artifacts"
+      exit 2
+  fi
+
+  PACKAGES_URLS="$(echo ${PACKAGES_METADATA}  | jq -r '.[].url')"
+  PACKAGE_URL=$(echo "${PACKAGES_URLS}" | egrep "${DISTRO}/.*${PACKAGE_NAME_REGEX}")
```

Comment:
`"${DISTRO}/.*${PACKAGE_NAME_REGEX}"`

Making `.*` not part of `PACKAGE_NAME_REGEX` looks too greedy to me and can match more unexpected, rather than help.
I'd prefer more explicit match to make sure we don't grab any accidental/temporary package name or dir, outside of expected one.

`tmp-st2-2.5dev-1.x86_64.rpm`, `test/st2_2.5dev-2_amd64.deb`